pages/phases/job-analysis/creating-assessments-from-competencies.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: article
-permalink: /hiring-phases/job-analysis/creating-a-structured-interview-from-competencies/
+permalink: /hiring-phases/job-analysis/creating-assessments-from-competencies/
 section: hiring-phases
-title: Creating a Structured Interview from Competencies
+title: Creating Assessments from Competencies
 sidenav: hiring-phases
 phase: job-analysis
 sub-phase-order: 060
-intro: Finish your workshop by breaking participants into groups to write questions for both the first and second phone assessment interviews. Use your defined competencies, proficiencies, and required experience to create two rounds of interview questions and good example responses.
+intro: Finish your workshop by breaking participants into groups to write questions for both the first and second assessment rounds. Use your defined competencies, proficiencies, and required experience to create guidance for SMEs reviewing applicant responses.
 ---
 
 <p class="usa-intro">
   {{ page.intro }}
 </p>
 
-As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. Two rounds of structured phone interviews, each conducted by a single SME, assess the breadth and then the depth of applicants' knowledge about your required competencies. Any applicant who doesn't receive a high enough score during the first interview will not progress to the second.
+As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. Two rounds of assessments can assess the breadth and then the depth of applicants' knowledge about your required competencies. Any applicant who doesn't receive a high enough score during the first assessment will not progress to the second.
```

Comment:
how about: 
"As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. In order to eliminate unqualified applicants between the two rounds, they should be measuring different things. For example, one interview can measure breadth of knowledge and the second interview can measure depth of knowledge.  Or a written demonstration project could measure technical competencies while a phone interview could measure competencies more easily assessed through oral communications. Therefore, any applicant who doesn't receive a high enough score during the first assessment will not progress to the second."

Reply:
awesome, I put this in prettymuch verbatim with a few tense changes and moving down a line from the intro:

As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. In order to eliminate unqualified applicants in each round, the two rounds should measure different things. For example, one interview can measure breadth of knowledge and the second interview can measure depth of knowledge. Or, a written demonstration project can measure technical competencies while a phone interview measures competencies more easily assessed through oral communications. Therefore, any applicant who doesn't receive a high enough score during the first assessment will not progress to the second. 

Use your defined competencies, proficiencies, and required experience to create guidance for SMEs reviewing applicant responses.